src/schema/types/group.graphql

```diff
@@ -0,0 +1,19 @@
+type Group {
+  id: ID!
+  product: Product!
+  type: TypeEnum!
+  minParticipants: Int!
+  maxParticipants: Int
+  name: String!
+  discount: String
```

Comment:
Isn't discount a number?

Reply:
Yes, also we don't need that name, it was a mistake

Reply:
Fixed